pkg/harvester/detail/harvesterhci.io.host/HarvesterHugepages.vue

```diff
@@ -0,0 +1,119 @@
+<script>
+import LabelValue from '@shell/components/LabelValue';
+import { HCI } from '../../types';
+import { DOC } from '../../config/doc-links';
+
+export default {
+  name:       'HarvesterHugepages',
+  components: { LabelValue },
+
+  props: {
+    node: {
+      type:     Object,
+      required: true,
+    },
+  },
+
+  computed: {
+    docsTransparentHugepagesLink() {
+      return DOC.TRANSPARENT_HUGEPAGES;
+    },
+  },
+
+  async fetch() {
+    const inStore = this.$store.getters['currentProduct'].inStore;
+
+    const hash = await this.$store.dispatch(`${ inStore }/findAll`, { type: HCI.HUGEPAGES });
+
+    this.hugepages = hash.find((node) => {
+      return node.id === this.node.id;
+    });
```

Comment:
Adding fallback empty object if can't find node id to prevent undefined error.
```suggestion
    }) || {}; 
``` 
<img width="1468" height="840" alt="image" src="https://github.com/user-attachments/assets/5b323a9f-41df-439a-abbb-cf3bee98f71f" />